frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -689,6 +689,9 @@ def get_attachments(self):
 		return attachments
 
 	def prepare_email_content(self):
+		email_account = self.get_outgoing_email_account()
+		if isinstance(self._bcc, list) and email_account.always_bcc:
```

Comment:
@0xD0M1M0 upon rethinking this, it isn't clear to me why we would only add `alyways_bcc` when `self._bcc` is a list. It seems like `self._bcc` could just as well be a string or None and in this case `alyways_bcc` would not be _always_. 🤔